helper/logformat/vault.go

```diff
@@ -15,6 +15,10 @@ import (
 const (
 	styledefault = iota
 	stylejson
+	// Setting a level for raw log output. While it's not technically a level, we need the ability
```

Comment:
You don't need any of this -- just use os.Stdout as the writer (https://golang.org/pkg/os/#pkg-variables)

Reply:
Right, that's what I did initially, but then I was reading [your comments](https://github.com/hashicorp/vault/pull/2195#issuecomment-269502197) on #2195 and thought that output from the audit-to-stdout and the primary vault logger would need to be synchronized (thinking vault logged to stdout). What needs to be changed for in #2195 for it to be merged?

Reply:
2195 will never be merged because it's a separate backend.